tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot

```diff
@@ -6,20 +6,65 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 Default Tags
 
 *** Test Cases ***
-Display HTML
+Get Login Page
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/authentication
+    Should contain  ${output}  <title>VCH Admin</title>
+
+While Logged Out Fail To Display HTML
     ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}
+    Should not contain  ${output}  <title>VIC: ${vch-name}</title>
```

Comment:
Added a check for the `302 Redirect` that sends the user to `/authentication` to each of these tests as a positive case.